### ISSUE TITLE
Display link to source code on webpage

### ISSUE THREAD

**Issue by moggers87** [xray7224/Pamrel#22]
I don't see one.

**Comment by xray7224** [comment]
I agree this is definitely an important thing that should be displayed. Do you think it's best to have a small footer at the bottom displayed on every page or some other way? Pamrel needs to retain it's no clutter UI but users needs to know where the source is.

**Comment by moggers87** [comment]
What about a link in the header?

**Comment by xray7224** [comment]
The top bar, maybe but not sure where it should best go when you're creating a new paste you currently have language dropdown, the pamrel name and the paste button and there is a possibility for more controls to come in the future such as those which would be required of #8. Given that Pamrel supports delete after a specific number of views and also delete after a specific date/time it might be nice for that to be exposed some how via the webUI. 

There is also the consideration of screen realestate not only on small laptops but small phone screens, possibly making #15 far harder than it is right now.
